tests/test_sklearn_voting_regressor_converter.py

```diff
@@ -0,0 +1,80 @@
+"""Tests GLMRegressor converter."""
+
+import unittest
+import numpy
+from sklearn import datasets
+from sklearn.linear_model import LinearRegression
+try:
+    from sklearn.ensemble import VotingRegressor
+except ImportError:
+    # New in 0.21
+    VotingRegressor = None
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeRegressor
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import (
+    FloatTensorType, Int64TensorType
+)
+from test_utils import dump_data_and_model
+
+
+def _fit_model(model, is_int=False):
```

Comment:
You may use https://github.com/onnx/sklearn-onnx/blob/d7a93a525c6ddfb8289a5c2d612755b8b9845885/tests/test_utils/tests_helper.py#L65, it does exactly the same thing.

Reply:
Done